src/reactions/index.ts

```diff
@@ -25,6 +25,8 @@ import deerSoundOgg from "../sound/reactions/deer.ogg?url";
 import deerSoundMp3 from "../sound/reactions/deer.mp3?url";
 import rockSoundOgg from "../sound/reactions/rock.ogg?url";
 import rockSoundMp3 from "../sound/reactions/rock.mp3?url";
+import waveSoundOgg from "../sound/reactions/wave.ogg?url";
```

Comment:
If the sound is your own, can you add a note in https://github.com/element-hq/element-call/blob/dg/add-wave-reaction/src/sound/LICENCE.md so we have our backs covered?

Reply:
9b42bc79dbd59be320540bdbb63b6064ae78452c